deploy/base/config.yaml

```diff
@@ -7,3 +7,4 @@ metadata:
     security-profiles-operator/config: ""
 data:
   SPOdImagePullPolicy: Always
+  EnableSelinux: "false"
```

Comment:
Can we directly use a bool here?
```suggestion
  EnableSelinux: false
```

Reply:
Haha, yes, I'll see about fixing this

Reply:
Hmm, I must be misunderstanding something. I'm getting errors trying to use a straight-up boolean in a configmap, seems like CMs only allow string values?